src/metrics/prometheus.go

```diff
@@ -20,6 +20,16 @@ var registerer = prometheus.WrapRegistererWith(prometheus.Labels{
 
 // Push performs a single push of all registered metrics to the pushgateway (if configured).
 func Push(config *core.Configuration) {
+	if family, err := prometheus.DefaultGatherer.Gather(); err == nil {
```

Comment:
Intended to keep in?

Reply:
Yep, I think it's nice to be able to read metrics from the end of a build log if you want. In this case, it's very hard to observe how many of these directories I'm creating/storing - it'd have to push it to Prometheus, then I'd have to go and query it back, just getting it out of the log file seems nice and easy.

We don't have very many metrics so it doesn't take much time or space in there.